gcp_variant_transforms/libs/beam_util.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2018 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Wrapper for Beam classes, e.g., Metrics.
```

Comment:
As mentioned above, I think we should limit the scope of this file. It's ok to have multiple files that specialize for each function.
Please also add an example of how classes in this file should be used as there are several classes and their connection is not trivial.